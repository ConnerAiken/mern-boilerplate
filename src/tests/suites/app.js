import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import App from "./../../client/app"; 

test('App component renders with h1 text', () => {
    const component = renderer.create(<App></App>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});