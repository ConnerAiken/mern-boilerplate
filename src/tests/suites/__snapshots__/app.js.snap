// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders with h1 text 1`] = `
<h1>
  Hello from React!
</h1>
`;
